### DISCUSSION TITLE
Add lists to GitHub.com mobile website

### DISCUSSION THREAD

**Discussion by aslafy-z** [community/community#38515] [Other Feature Feedback, Questions, & Ideas]
Lists are shown on GitHub.com desktop website and on GitHub Mobile app, not on GitHub.com mobile website.
Please add them.

**Comment by david-engelmann** [comment]
Your discussion are also not on GitHub.com mobile

**Comment by ossanna16** [comment]
Hi there @aslafy-z and welcome to our community! Thank you for the feedback!

**Comment by jfullstackdev** [comment]
I'm not so sure about that, can you share the screenshot ? 

it should work the same thing as GitHub Web because you are using the browser

**Reply by aslafy-z** [reply]
Here you go:

not starred: 
![Screenshot_20221226-140218](https://user-images.githubusercontent.com/8191198/209551969-7859341c-a9aa-4a49-8795-ffeacbb0f237.png)
Then starred: 
![Screenshot_20221226-140223](https://user-images.githubusercontent.com/8191198/209552000-4b839f0e-40c6-4d80-92a4-68292648e46a.png)

**Reply by aslafy-z** [reply]
And on desktop:

Not starred: ![Screenshot_20221226-140409.png](https://user-images.githubusercontent.com/8191198/209552069-e38c34d5-af0d-417d-b073-39380d7bab1c.png)
Lists appears: ![Screenshot_20221226-140420.png](https://user-images.githubusercontent.com/8191198/209552102-f0dfb0e4-e32d-4ea9-a15d-ee1aeefb3e6f.png)
